src/views/UrbitID/ResetKeys/ResetExecute.tsx

```diff
@@ -127,10 +127,11 @@ export default function ResetExecute({
   );
 
   const performReticket = useCallback(async () => {
+    const { isConnected } = useWalletConnect();
```

Comment:
shouldnt this be outside of the useCallback hook, with `isConnected` in the useCallback dependency array?